tox.ini

```diff
@@ -13,12 +13,11 @@ deps =
 
     dj111: Django>=1.11,<2.0
     dj20: Django>=2.0,<2.1
+    dj22: Django>=2.2,<3.0
 
     cms40: https://github.com/divio/django-cms/archive/release/4.0.x.zip
 
 basepython =
-    py27: python2.7
```

Comment:
We will need to get the removal of 2.7 agreed by Divio, it's not our call to remove this.